### PR TITLE
Added Urls

### PR DESCRIPTION
**Pull request by MacSplody**
- Unofficial Skyrim Special Edition Patch
- unofficial skyrim survival patch
- MoonAndStar
- Undeath
- Undeath Classical Lichdom
- Improved closefaced helmets
- Morrowloot Ultimate
- SMIM
- TouringCarriages
- Cutting Room Floor
- BetterQuestObjectives
- Skyrim Hair Replacer
- Better Vampires
- Riftrun
- Dovahkiin Retreat
- Exe Boss Mods
- Open Cities Skyrim
- Alternate Start
- Weapons of the Third Era
- High Level Enemies
- Skyrim Alchemy Fixed
- Warmonger Armory
- Immersive Armors
- Rich Skyrim Merchants
- Morrowloot
- JK''s Riverwood
- JKs Whiterun

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -716,60 +742,74 @@ plugins:
...
   - name: 'Skyrim Alchemy Fixed.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2262/']
     tag:
       - Delev
   - name: 'Skyrim Alchemy Fixed - Ordinator.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2262/']
     tag:
       - Delev
   - name: 'aMidianBorn_ContentAddon.esp'
+    url: ['https://www.nexusmods.com/skyrim/mods/24909/']
```
Comment (MacSplody) [inline masterlist.yaml:778]: Oldrim mod, not sure what the stance is on these being added. I don't think this will ever be ported over officially.

**Comment by MacSplody** [comment]
I'll leave any comments with mod names for a format pull.
Is it preferable to add the Name comment above the esp just after?

File: masterlist.yaml (hunk 1 above)
Comment (Ortham) [reply masterlist.yaml:778]: My stance is that if people are using it in SE, it can go in the SE masterlist.

**Comment by Ortham** [comment]
> Is it preferable to add the Name comment above like this?

I would put it in the location metadata, e.g.

```
- name: 'BSAssets.esm'
  url: 
    - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/10917/'
      name: 'Beyond Skyrim - Bruma SE on Nexus Mods'
```

It's a little more verbose, but makes that information accessible as metadata.

**Comment by MacSplody** [comment]
Yeah that looks much better.

File: masterlist.yaml (hunk 1 above)
Comment (MacSplody) [reply masterlist.yaml:778]: And It's okay to use the oldrim link if no SE version has been released?

Comment (Ortham) [reply masterlist.yaml:778]: Yes, that's where the plugin is found, after all.

**Pull request merged**